packages/wonder-blocks-icon-button/components/icon-button.js

```diff
@@ -116,6 +116,10 @@ export type SharedProps = {|
  * light (whether the IconButton will be rendered on a dark background),
  * disabled , test ID, and custom styling.
  *
+ * Note: the size of an IconButton match the size of icon it wraps which is
+ * 24x24 pixels.  The focus ring which is display on hover and focus is much
+ * large but does not affect its size.  This matches the behavior of Button.
```

Comment:
Grammar nits: "match**es** the size of **the** icon"; "which is display**ed** on hover", "much large**r** but"

Reply:
Thanks.  I think I need grammarly. 😞